pinc/charsuite-basic-latin.inc

```diff
@@ -6,7 +6,7 @@ $charsuite->codepoints = [
     # https://en.wikipedia.org/wiki/Basic_Latin_(Unicode_block)
     'U+0020-U+007e',
     # https://en.wikipedia.org/wiki/Latin-1_Supplement_(Unicode_block)
-    # without the soft hyphen
+    # without the soft hyphen and micro sign
```

Comment:
Need to remove reference to removing micro sign from character suite -- it was only removed from the pickers.

Reply:
Done.